File: src/0231-Power-of-Two/power_of_two.cpp
#include <iostream>

class Solution {
public:
    bool isPowerOfTwo(int n) {
        return n > 0 && (n & (n - 1)) == 0;
    }
};

int main() {
    int n = 16;
    
    Solution solution;

    std::cout << solution.isPowerOfTwo(n);
    
    return 0;
}

File: src/0231-Power-of-Two/power_of_two.py
class Solution(object):
    def isPowerOfTwo(self, n: int) -> bool:
       return n > 0 and (n & (n - 1)) == 0

if __name__ == "__main__":
    n: int = 16
        
    solution = Solution()
    
    print(solution.isPowerOfTwo(n))